Fix build with LLVM 19

../src/math/lp/static_matrix.h:83:72: error: no member named 'get' in 'static_matrix<T, X>'; did you mean 'set'?
   83 |         ref operator=(ref & v) { m_matrix.set(m_row, m_col, v.m_matrix.get(v.m_row, v.m_col)); return *this; }
      |                                                                        ^~~
      |                                                                        set

Index: core/src/math/lp/static_matrix.h
--- core/src/math/lp/static_matrix.h.orig
+++ core/src/math/lp/static_matrix.h
@@ -80,8 +80,6 @@ class static_matrix (public)
         ref(static_matrix & m, unsigned row, unsigned col):m_matrix(m), m_row(row), m_col(col) {}
         ref & operator=(T const & v) { m_matrix.set( m_row, m_col, v); return *this; }
 
-        ref operator=(ref & v) { m_matrix.set(m_row, m_col, v.m_matrix.get(v.m_row, v.m_col)); return *this; }
-
         operator T () const { return m_matrix.get_elem(m_row, m_col); }
     };
 
